Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -702,7 +769,12 @@ public void CLIScriptRegressionTest()
 
             // Act
             CLIProcessor CLI = new CLIProcessor();
-            CLI.ExecuteArgs(new string[] { "script", "-f" , scriptFile });
+         
+
+            Task.Run(async () =>
+            {
+                await CLI.ExecuteArgs(new string[] { "script", "-f", scriptFile });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)